docs/prometheus.md

```diff
@@ -0,0 +1,329 @@
+It's possible that you already have some part of the K8s Prometheus stack already installed, and would like to make use of it. This document describes how to deal with all the possible cases.
+
+__NOTE:__ In this document we assume that `${NAMESPACE}` represents namespace in which the Sumo Logic Kubernetes Collection is going to be installed.
+
+- [No Prometheus in the cluster](#no-prometheus-in-the-cluster)
+- [Prometheus Operator in the cluster](#prometheus-operator-in-the-cluster)
+
+  - [Custom Resource Definition compatibility](#custom-resource-definition-compatibility)
+
+  - [Installing Sumo Logic Prometheus Operator side by side with existing Operator](#installing-sumo-logic-prometheus-operator-side-by-side-with-existing-operator)
+
+    - [Set Sumo Logic Prometheus Operator to observe installation namespace](#set-sumo-logic-prometheus-operator-to-observe-installation-namespace)
+
+  - [Using existing Operator to create Sumo Logic Prometheus instance](#using-existing-operator-to-create-sumo-logic-prometheus-instance)
+
+    - [Disable Sumo Logic Prometheus Operator](#disable-sumo-logic-prometheus-operator)
+
+  - [Prepare Sumo Logic Configuration to work with existing Operator](#prepare-sumo-logic-configuration-to-work-with-existing-operator)
+
+  - [Using existing Kube Prometheus Stack](#using-existing-kube-prometheus-stack)
+
+    - [Build Prometheus Configuration](#build-prometheus-configuration)
+
+## No Prometheus in the cluster
+
+If you don't have Prometheus or Kube Prometheus Stack installed in your cluster
+there is not much you need to worry about.
+There is no special configuration required, unless you want to have
+[Custom Application Metrics](/docs/collecting-application-metrics.md) or
+[Custom Kubernetes Metrics](/docs/collecting-kubernetes-metrics.md),
+but these steps can be performed after initial installation.
+
+## Prometheus Operator in the cluster
+
+If you already have Prometheus operator in your cluster, the necessary changes depend on the compatibility between Prometheus Operator used in your cluster and Prometheus Operator used by Sumo Logic Kubernetes Collection.
+
+In that situation we support three scenarios:
+
+- Installing Sumo Logic Prometheus Operator side by side with existing Operator
+
+  __NOTE:__ This is possible if Custom Resource Definitions in the cluster are compatible with those required by Sumo Logic,
+  or there is possibility to install Custom Resource Definition which will be accepted by all
+  Prometheus Operators
+- Using existing Operator to create Sumo Logic Prometheus instance
+
+  __NOTE:__ This is possible if Custom Resource Definitions in the cluster are compatible with those required by Sumo Logic,
+  or there is possibility to install Custom Resource Definition which will be accepted by all
+  Prometheus Operators
+
+- Using existing Kube Prometheus Stack
```

Comment:
```suggestion
1. Installing Sumo Logic Prometheus Operator side by side with existing Operator

   __NOTE:__ This is possible if Custom Resource Definitions in the cluster are compatible with those required by Sumo Logic,
   or there is possibility to install Custom Resource Definition which will be accepted by all
   Prometheus Operators

1. Using existing Operator to create Sumo Logic Prometheus instance

   __NOTE:__ This is possible if Custom Resource Definitions in the cluster are compatible with those required by Sumo Logic,
   or there is possibility to install Custom Resource Definition which will be accepted by all
   Prometheus Operators

1. Using existing Kube Prometheus Stack
```

1. Installing Sumo Logic Prometheus Operator side by side with existing Operator

   __NOTE:__ This is possible if Custom Resource Definitions in the cluster are compatible with those required by Sumo Logic,
   or there is possibility to install Custom Resource Definition which will be accepted by all
   Prometheus Operators

1. Using existing Operator to create Sumo Logic Prometheus instance

   __NOTE:__ This is possible if Custom Resource Definitions in the cluster are compatible with those required by Sumo Logic,
   or there is possibility to install Custom Resource Definition which will be accepted by all
   Prometheus Operators

1. Using existing Kube Prometheus Stack